src/common/runtime/src/bin.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::Parser;
+
+#[derive(Debug, Default, Parser)]
+pub struct Command {
+    #[arg(long, action = clap::ArgAction::Count)]
+    test_diff_priority_cpu: u8,
+}
+
+fn main() {
+    let cmd = Command::parse();
+
+    // make sure one and only one of them is set
+    let cmds_state = [(cmd.test_diff_priority_cpu > 0, "test-diff-priority-cpu")];
```

Comment:
Why do we need this cmd argument? An argument to control the times of the loop to run the test in `test_spec_priority_and_workload()` is more helpful than this argument. For example, a `num_loops` argument.

Reply:
Maybe we will add multi-command selection in the future, is there better solution?